### ISSUE TITLE
cef on older windows versions

### ISSUE THREAD

**Issue by memejangOFFICIAL** [SpiralP/classicube-cef-plugin#58]
When my friends install cef(they use older windows versions) it shows up with error 126 even though its named ClassiCube.exe. does it not support these versions?

**Comment by imthehappy** [comment]
Yeah tell us why error 126 shows up even Its named ClassiCube.exe when using cef older windows versions

**Comment by SpiralP** [comment]
126 looks like this? "Error 126: Specified module could not be found", so a dll dep is probably missing on the OS you're using

you can use https://github.com/lucasg/Dependencies to see the dependency tree of executables on windows, so you can probably learn what is missing if you put `libcef.dll` or the plugin dll into it

i use the latest windows version and i don't think it has this issue (yet?)

**Comment by Alexcrishalt** [comment]
i hate my life now i have more issues

**Comment by Alexcrishalt** [comment]
Seriously you need visual studio dummies

**Comment by Alexcrishalt** [comment]
I will delete sucker

**Comment by SpiralP** [comment]
i only use (and want to support) the latest version of windows right now

reopen with more info if this is still an issue
